cmd/owl/main.go

```diff
@@ -196,6 +203,10 @@ func main() {
 
 		jobs := make(chan owl.Job, 10)
 
+		jobs <- &WatcherJob{
```

Comment:
We are using the same job at two places. Can we define it at one place and just pass the reference each time to the channel?